docs/reference/rest-api/index.asciidoc

```diff
@@ -54,7 +53,9 @@ not be included yet.
 * <<snapshot-lifecycle-management-api,Snapshot lifecycle management APIs>>
 * <<sql-apis,SQL APIs>>
 * <<synonyms-apis,Synonyms APIs>>
+* <<text-structure-apis,Text structure APIs>>
 * <<transform-apis,{transform-cap} APIs>>
+* <<test-grok-pattern,Test Grok pattern API>>
```

Comment:
I don't think this line is needed if you included the text structure APIs in line 56 as it will include the Test grok pattern API, too.

```suggestion
```

Reply:
fixed